src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -59,7 +66,8 @@ export default class Message extends React.Component<Props, any> {
             <SenderName>{this.props.message.senderName}</SenderName>
             <MetadataText>{this.props.message.time}</MetadataText>
           </Header>
-          <BodyText>{this.props.message.body}</BodyText>
+          {this.props.artworkPreview && <ArtworkPreviewContainer>{this.props.artworkPreview}</ArtworkPreviewContainer>}
+          <BodyText>{this.props.message.body.split("\n\nAbout")[0]}</BodyText>
```

Comment:
This is just a really rudimentary clean-up of the initial message so it's easier to see what things should actually look like. We'll eventually do real message sanitizing in metaphysics.